### DISCUSSION TITLE
Need help with the self-hossting instructions

### DISCUSSION THREAD

**Discussion by ovizii** [lukevella/rallly#893] [Q&A] [answered]
I am wondering why these instructions start with: 

> The best way to self-host Rallly is using the [official Docker image](https://hub.docker.com/r/lukevella/rallly). This image contains a build that is specifically intended for self-hosting.

But then go on to "git clone" and use a docker-compose.yml which seems to build a local image or am I missing something here?

**Comment by lukevella** [comment, marked as answer]
Hi @ovizii, if you follow [these instructions](https://support.rallly.co/self-hosting/docker-compose) it explains how to use the docker compose config in the [rallly-selfhosted repo](https://github.com/lukevella/rallly-selfhosted). This config uses the pre-built image from docker hub.

**Reply by ovizii** [reply]
I'm sorry I forgot to follow up on this discussion, I finally ended up finding those links and managed to get everything up and running.

**Comment by pnoelfr** [comment]
Good morning

To host Rally: can you give us the information to size our servers

We have almost 10,000 users


Number of CPUs
ram size
system size
database size


THANKS

**Comment by danielwagn3r** [comment]
I started with an Azure B2s machine running Ubuntu Server 22.04

Additionally to the provided compose file I use a combination of nginx and certbot, both dockerized as well, for the reverse proxy and certificate handling.

And https://github.com/containrrr/watchtower for automatic updates, I recommend to use a different label than`latest` in this case.
